pages/common/youtube-viewer.md

```diff
@@ -0,0 +1,20 @@
+# youtube-viewer
+
+> Command-line application for searching and playing videos from YouTube.
+> More information: <https://github.com/trizen/youtube-viewer>.
+
+- Search for a video:
+
+`youtube-viewer {{search_term}}`
+
+- Login to your YouTube account:
+
+`youtube-viewer --login`
+
+- Watch a video with a specific URL in VLC:
+
+`youtube-viewer --player={{vlc}} {{https://youtube.com/watch?v=1dastnond}}`
+
+- Get to a search prompt and play the selected video in 720p:
```

Comment:
I'm confused by this. What does `Get to a search prompt` mean specifically?

Reply:
Something like this:
![image](https://user-images.githubusercontent.com/58830309/84422445-05220000-abeb-11ea-8fe6-afc39ca03464.png)

Reply:
@sbrl Does that clarify?

Reply:
Ah, I see! Sorry, this PR slipped under the radar.  It seems like this comment has been resolved further down.